poetry/console/commands/run.py

```diff
@@ -28,7 +28,15 @@ def handle(self) -> Any:
         if scripts and script in scripts:
             return self.run_script(scripts[script], args)
 
-        return self.env.execute(*args)
+        try:
+            return self.env.execute(*args)
+        except FileNotFoundError:
+            self.line_error("")
+            self.line_error(
+                "<error>Error: Command {} is not found.</error>".format(script)
+            )
+            self.line_error("")
```

Comment:
```suggestion
            self.line_error(f "<error><c1>{script}</c1>: command not found.</error>")
```

Maybe something more concise is better?